deploy/docker-compose/docker-compose-traffic-replay.yml

```diff
@@ -2,4 +2,7 @@ version: '3'
 services:
   traffic-replay:
     image: "ddtraining/traffic-replay:latest"
+    environment:
+      - FRONTEND_HOST
+      - FRONTEND_PORT
```

Comment:
I believe just these lines should be enough, and changing the Dockerfile isn't necessary.